content/docs/components/hyperparameter-tuning/hyperparameter.md

```diff
@@ -109,7 +109,17 @@ Run the following command to launch an experiment using the random algorithm
 example:
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha3/random-example.yaml
+curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha3/random-example.yaml --output random-example.yaml
```

Comment:
This command doesn't actually launch the experiment, but the introductory sentence implies that it does.

A quick fix is to change "command" to "commands" on line 108.

The longer fix, which perhaps results in a clearer doc, is to change line 108 to this: "Run the following commands to launch an experiment using the random algorithm example:" and then make the subsequent points into a numbered list.

Reply:
Done, PTAL

Reply:
Looks great!